.gitignore

```diff
@@ -15,3 +15,6 @@
 .idea
 .*.swp
 .*.swo
+
+# Ignore database configuration
```

Comment:
Very Nice. Following the tips of @hosseintoussi, i believe this will not be necessary.